daml-assistant/daml-helper/src/DA/Daml/Helper/Main.hs

```diff
@@ -340,6 +343,12 @@ commandParser = subparser $ fold
                 , clientMetadataPlugin = Nothing
                 }
 
+    httpJsonFlag :: ShowJsonApi -> Parser LedgerApi
+    httpJsonFlag (ShowJsonApi showJsonApi) = flag GRPC HttpJson $
+        (if showJsonApi then mempty else internal)
+        <> long "http-json-api"
```

Comment:
given that the command is called `daml json-api` not `daml http-json-api` how do you feel about calling this `--json-api`?

Reply:
sure.